pkg/csi/service/common/util.go

```diff
@@ -351,6 +351,10 @@ func ConvertVolumeHealthStatus(volHealthStatus string) (string, error) {
 	case string(pbmtypes.PbmHealthStatusForEntityUnknown):
 		return string(pbmtypes.PbmHealthStatusForEntityUnknown), nil
 	default:
-		return "", fmt.Errorf("cannot convert invalid volume health status %s", volHealthStatus)
+		// NOTE: volHealthStatus is not set by SPBM in this case.
+		// This implies the volume does not exist any more.
```

Comment:
Volume not existing is different to saying the volume health is inaccessible. Isn't this a bug in SPBM?

Reply:
SPBM is the component that should report the volume health correctly. I did not quite understand why SPBM should not be fixed.

Reply:
Here are the notes after the offline meeting on this topic. We agreed that the volume health can be set to "inaccessible" whenever CNS says it is "red" or "" (missing). PSP operators expect the Volume health to be eventually consistent. They also have the health timestamp to allow waiting for a "long" time (like an hour) before taking corrective actions based on the Volume health. So it is a correct design choice to reflect the current CNS health snapshot on the PVCs.